src/OpenIddict.Core/OpenIddictOptions.cs

```diff
@@ -16,9 +16,6 @@ namespace OpenIddict {
     /// </summary>
     public class OpenIddictOptions : OpenIdConnectServerOptions {
         public OpenIddictOptions() {
-            AuthorizationEndpointPath = IntrospectionEndpointPath = LogoutEndpointPath =
```

Comment:
FYI, "nullifying" the endpoint paths is no longer necessary with ASOS beta7 because all the endpoints are disabled by default.